### ISSUE TITLE
How to disable triggering the picker on span click

### ISSUE THREAD

**Issue by pepijnolivier** [atais/angular-eonasdan-datetimepicker#18]
My HTML currently looks like the below. As you can see, I've added two `input-group-addons` with which the user will be able to go to the next / previous day without opening the datepicker.

```
<div class="input-group">
    <span class="input-group-addon previous-day" ng-click="addDays(-1, $event); ">
        <span class="fa fa-angle-left"></span>
    </span>
    <input class="form-control" datetimepicker ng-model="value" options="options" on-change="onChanged()" />
    <span class="input-group-addon next-day" ng-click="addDays(1, $event)">
        <span class="fa fa-angle-right"></span>
    </span>
    <span class="input-group-addon">
        <span class="fa fa-calendar"></span>
    </span>
</div>
```

**This is how I'm setting the date to the next / previous day:**
```
$scope.addDays = function(howMany, $event) {
    if($scope.options.date) {
        $scope.options.date = $scope.options.date.add(howMany, 'days');
    }
}

```
**The problem**
When I click on the span, the datepicker is opened, which it is expected to do for the addon with the calendar icon, but obviously I don't want it to open when the user clicks on one of the arrows.

**What I've already tried**

- $event.preventDefault()
- $event.stopPropagation();
- Get the datepicker instance and call `dtpInstance.hide();` right after changing date
- Hacky stuff like `$('.bootstrap-datetimepicker-widget').css('display', 'none');` and `$('.bootstrap-datetimepicker-widget').css('display', 'block');`

None of the above gave me the result I expected.

----

How can I do this the right way?

**Comment by atais** [comment]
have you tried option mentioned here: https://github.com/atais/angular-eonasdan-datetimepicker/issues/10 ?

```allowInputToggle: false```

**Comment by pepijnolivier** [comment]
@atais Just tried it,  but as the name suggests, that option only has an impact on the `input`, not on the `input-addons`.

Really, I'd need an option like `allowInputAddonToggle: false`, but sadly there is no such option.

I just realized, since all this functionality is handled by the eonasdan datetimepicker, I should probably move this issue over there.

JSFiddle: https://jsfiddle.net/gjggqvd2/

**Comment by pepijnolivier** [comment]
@atais FYI, I fixed the issue as follows (got the idea via https://github.com/Eonasdan/bootstrap-datetimepicker/issues/48 )

---


1. **Use a custom class for the extra addons**
I chose `custom-input-group-addon`. Make sure you're not using the `input-group-addon` class.
This already prevents the datepicker from being shown on click, but now the styling is off.

2. **Extend your styling**
I'm currently using sass for all styling. This is how I added al the necessary style to the custom class:
```
.custom-input-group-addon {
    @extend .input-group-addon;
}
```


So now, the issue is indeed resolved ;-)
